pyproject.toml

```diff
@@ -61,18 +61,20 @@ build.targets.sdist.include = ["/src", "/tests","tox.ini"]
 version.source = "vcs"
 
 [tool.ruff]
-select = ["ALL"]
 line-length = 120
-target-version = "py38"
-isort = {known-first-party = ["pyproject_fmt"], required-imports = ["from __future__ import annotations"]}
+src = ["src", ""]
+
+[tool.ruff.lint]
+select = ["ALL"]
+isort.required-imports = ["from __future__ import annotations"]
```

Comment:
I strongly prefer to inline things rather than define new tables.

Reply:
Okay.

Reply:
I wasn't sure, per-file-ignores is a separate table already.